src/PDFsharper.UnitTests/Drawing/FontHelperTests.cs

```diff
@@ -370,5 +370,187 @@ public void CreateStyle_IsBold_IsItalic()
 
             Assert.IsTrue(fontStyle == XFontStyle.BoldItalic, "fontStyle should be BoldItalic");
         }
+
+        #region Font name mapping tests
```

Comment:
I'll move them to a new test class for xfont, i wanted to call it through xfont to be sure it was calling the method to assign the content font name.